Unix/configeditor/configeditor.cpp

```diff
@@ -372,7 +392,10 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         // If property isn't the one we want, move along
         if (prop != arg1)
         {
-            cout << line << std::endl;
+            if (!fDontEchoFile)
```

Comment:
@jeffaco 
Added the flag to determine output to stdout.

I did not chose the current behavior of redirecting from stdin to stdout for following reason:
1. My current implementation exits as soon as the property in query is reached. This would mean that complete omiserver.conf might not get redirected.
2. As of now query op outputs string like **<property> has value <value>** or **<property> does not have value <value>** . With this I intended that someone running the command manually can get to know the result without looking at the exit code.  Hence if stdout is redirected to /dev/null these strings too will not get displayed.

Let me know if this makes sense or I should change my implementation to redirect complete file to stdout for query operation to.